.prettierignore

```diff
@@ -0,0 +1,16 @@
+.github/
```

Comment:
This looks like it needs to stay consistent with the `.gitignore` as well.  Let's drop a comment in here that says to update the `.gitignore` and vice versa, so we remember.  Our convention for these types of comments is:

```
# Heads up!
# Some important message.
```